weather_mv/loader_pipeline/ee.py

```diff
@@ -520,18 +521,22 @@ def process(self, uri: str) -> t.Iterator[AssetData]:
         queue = Queue(maxsize=1)
         process = Process(target=self.convert_to_asset, args=(queue, uri))
         process.start()
-        kill_subprocess = False
 
-        while not kill_subprocess:
+        while True:
             if not queue.empty():
                 asset_data = queue.get_nowait()
 
-                if asset_data is not None:
-                    yield asset_data
-                else:
-                    kill_subprocess = True
+                # Not needed now but keeping this check for backwards compatibility.
```

Comment:
Why is this not needed?

Reply:
Now we are checking below if the process is alive or not and the loop will break when the process finishes.